dex/pair/src/safe_price.rs

```diff
@@ -91,188 +61,103 @@ pub trait SafePriceModule:
     + permissions_module::PermissionsModule
     + pausable::PausableModule
 {
-    #[endpoint(updateAndGetTokensForGivenPositionWithSafePrice)]
-    fn update_and_get_tokens_for_given_position_with_safe_price(
-        &self,
-        liquidity: BigUint,
-    ) -> MultiValue2<EsdtTokenPayment<Self::Api>, EsdtTokenPayment<Self::Api>> {
-        self.update_safe_state_on_the_fly();
-
-        let c_state = self.get_current_state_or_default();
-        let total_supply = self.lp_token_supply().get();
-        let first_token_id = self.first_token_id().get();
-        let second_token_id = self.second_token_id().get();
-        let big_zero = BigUint::zero();
-
-        let pool_initialized = total_supply != big_zero
-            && c_state.first_token_reserve_weighted != big_zero
-            && c_state.second_token_reserve_weighted != big_zero;
-
-        let (first_token_worth, second_token_worth) = if pool_initialized {
-            let first_worth = &liquidity * &c_state.first_token_reserve_weighted / &total_supply;
-            let second_worth = &liquidity * &c_state.second_token_reserve_weighted / &total_supply;
-
-            (first_worth, second_worth)
-        } else {
-            (big_zero.clone(), big_zero)
-        };
-
-        MultiValue2::from((
-            EsdtTokenPayment::new(first_token_id, 0, first_token_worth),
-            EsdtTokenPayment::new(second_token_id, 0, second_token_worth),
-        ))
-    }
-
-    #[endpoint(updateAndGetSafePrice)]
-    fn update_and_get_safe_price(
+    #[endpoint(updateSafePriceInfo)]
+    fn update_safe_price_info(
```

Comment:
rename this endpoint. setSafePriceParams

Reply:
Updated.